hack/generated/controllers/recordings/Test_CosmosDB_CRUD.yaml

```diff
@@ -172,14 +172,14 @@ interactions:
       - application/json
       Test-Request-Attempt:
       - "0"
-    url: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/asotest-rg-dnhfns/providers/Microsoft.Resources/deployments/k8s_f9f7e648-5320-5c32-8775-febefed1fa3a?api-version=2019-10-01
+    url: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/asotest-rg-dnhfns/providers/Microsoft.Resources/deployments/k8s_f9f7e648-5320-5c32-8775-febefed1fa3a?api-version=2019-10-01
```

Comment:
What's the canonical form here? Was `resourcegroups` wrong or is `resourceGroups` now wrong?

Reply:
To clarify - I think we should be using the canonical forms of the ARM URIs even if either case is supported.

Reply:
I'm not actually 100% sure there is an _official_ cannonicalized form which is used consistently across Azure.

With that said, the [ARM docs](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#resourceid) use `resourceGroups`, and other providers such as `Microsoft.Storage` seem to use `storageAccounts` (not `storageaccounts`), so I think it makes sense to use that form as it's:
1. More readable.
2. The one used at least in the official docs I've seen.

Reply:
Works for me. 👍